snmp/datadog_checks/snmp/data/default_profiles/alcatel-lucent-omni-access-wlc.yaml

```diff
@@ -0,0 +1,70 @@
+extends:
+  - alcatel-lucent.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.6486.800.1.1.2.2.2.*
+metrics:
+  - MIB: WLSX-SWITCH-MIB
+    table:
+      name: wlsxSysXProcessorTable
+      OID: 1.3.6.1.4.1.14823.2.2.1.1.1.9
+    symbols:
+      - name: cpu.usage
+        OID: 1.3.6.1.4.1.14823.2.2.1.1.1.9.1.3
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.14823.2.2.1.1.1.9.1.2
+          name: sysXProcessorDescr
+        tag: sys_x_processor_descr
```

Comment:
Same as here, we need to tag by at least `cpu`: https://github.com/DataDog/integrations-core/pull/15094/files#r1247835716

Reply:
If possible, maybe we can use the index as cpu tag.

Reply:
In this case, the index seems to be sysXProcessorID and could be suitable as cpu tag.

Maybe we can also have a `sys_x_processor_id` tag.

Reply:
Done thank you!